tests/promptlab/test_promptlab_model.py

```diff
@@ -1,12 +1,25 @@
 from unittest import mock
 
 import pandas as pd
+import pytest
 
 from mlflow._promptlab import _PromptlabModel
 from mlflow.entities.param import Param
 
 
-def test_promptlab_prompt_replacement():
+@pytest.fixture
+def Model():
+    def construct_model(route):
+        prompt_parameters = [Param(key="thing", value="books")]
+        model_parameters = [Param(key="temperature", value=0.5), Param(key="max_tokens", value=10)]
+        prompt_template = "Write me a story about {{ thing }}."
+
+        return _PromptlabModel(prompt_template, prompt_parameters, model_parameters, route)
+
+    return construct_model
```

Comment:
```suggestion
def construct_model(route):
        prompt_parameters = [Param(key="thing", value="books")]
        model_parameters = [Param(key="temperature", value=0.5), Param(key="max_tokens", value=10)]
        prompt_template = "Write me a story about {{ thing }}."

        return _PromptlabModel(prompt_template, prompt_parameters, model_parameters, route)

```

can we just defne `construct_model`? We don't need to fixturize everything.

Reply:
this is true haha